mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -325,6 +325,50 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 				Description: `When used with wait_for_instances specifies the status to wait for. When STABLE is specified this resource will wait until the instances are stable before returning. When UPDATED is set, it will wait for the version target to be reached and any per instance configs to be effective and all instances configs to be effective as well as all instances to be stable before returning.`,
 				<% end -%>
 			},
+			<% unless version == "ga" -%>
+			"stateful_internal_ip": {
```

Comment:
@rileykarson Just want to double-check that when you say "no update support" you mean "we ship it as-is & if they want to remove nics they do it manually" rather than "we make these new fields ForceNew for now and change that to allow in-place updates later."

Reply:
No update support means the second: "we make these new fields ForceNew for now and change that to allow in-place updates later."

Reply:
So based on the current PR, we add **ForceNew: true** to the statefulIps, and in another PR, we allow in-place updates.

Reply:
@hao-nan-li yep.

Reply:
Just want to make sure, it's an update test required if we set the field to forceNew?

Reply:
no, that's not required. in fact the fields need to actively _not_ be changed in an update test - otherwise the test will pass but not exercise the update-in-place path.

If the resource doesn't already have it, you could consider using lifecycle.prevent_destroy to ensure that the test doesn't accidentally delete the resource: https://github.com/hashicorp/terraform-provider-google/issues/12594